docs/specs/encoding.md

```diff
@@ -1,18 +1,24 @@
 # Encoding
 
-The existing .NET encoding APIs (System.Text.Encoding) don't work super well in non-allocating data pipelines:
+The existing .NET encoding APIs (System.Text.Encoding) don't work super well in
+non-allocating data pipelines:
```

Comment:
What is the line length you are using here? What do we get from standardizing it (just curios)?

Reply:
Sure!  The standard seems to be 80 characters in most repositories.  This helps keep the line length readable on older devices and devices with smaller screens.  Some editors (like Atom) will default the line length to 80.